src/lsptoolshost/languageStatusBar.ts

```diff
@@ -40,3 +41,43 @@ class WorkspaceStatus {
         });
     }
 }
+
+class ProjectContextStatus {
+    static createStatusItem(
+        context: vscode.ExtensionContext,
+        languageServer: RoslynLanguageServer,
+        languageServerEvents: RoslynLanguageServerEvents
+    ) {
+        const projectContextService = languageServer._projectContextService;
+
+        const item = vscode.languages.createLanguageStatusItem(
+            'csharp.projectContextStatus',
+            languageServerOptions.documentSelector
+        );
+        item.name = vscode.l10n.t('C# Project Context Status');
+        item.detail = vscode.l10n.t('Project Context');
```

Comment:
Instead of 'Project Context' in the right hand part of the status bar item, maybe 'Active File Context'?

Reply:
I'll change to that. @mikadumont is there a name you prefer here?